webview-ui/vite.config.ts

```diff
@@ -54,7 +54,18 @@ const persistPortPlugin = (): Plugin => ({
 export default defineConfig(({ mode }) => {
 	let outDir = "../src/webview-ui/build"
 
-	const pkg = JSON.parse(fs.readFileSync(path.join(__dirname, "..", "src", "package.json"), "utf8"))
+	// kilocode_change start - read package.json fresh every time to avoid caching issues
```

Comment:
That's not what the change is, is it? It just adds a catch.

Reply:
This is to prevent the cache of the package.json from the vite hot reload (Only affect to the local dev env)

Reply:
the change looks scary -- we can accidentally push a version 0.0.0 now?